modules/distr-tracing-config-otel-send-data-to-monitoring-stack.adoc

```diff
@@ -0,0 +1,37 @@
+////
+This module included in the following assemblies:
+-distr_tracing_install/distributed-tracing-deploying-otel.adoc
+////
+:_content-type: REFERENCE
+[id="distr-tracing-config-otel-collector_monitoring_{context}"]
+= Sending metrics to the monitoring stack
+
+.Sample `+PodMonitor+` that configures monitoring stack to scrape OpenTelemetry collector and removes duplicated labels.
```

Comment:
Make this into a paragraph that explains the example instead of an extra long heading, e.g. something like:

```
The following example shows how to configure the monitoring stack to do X thing:
```